api/src/main/java/org/pmiops/workbench/cohorts/CohortService.java

```diff
@@ -82,12 +128,35 @@ public TerraJob submitGenomicsCohortExtractionJob(String workspaceNamespace, Str
                 new FirecloudMethodConfiguration()
                     .deleted(false)
                     .inputs(
-                        createInputParameter(
-                            "Hello from AoU (" + workspaceNamespace + "/" + workspaceName + ")!"))
+                        new ImmutableMap.Builder<String, String>()
```

Comment:
The final output should be into the user's bucket. Is that out of scope for this ticket? If so, I would just put some TODO here or something to model the current state of the workflow.